src/ErrorMiddleware.php

```diff
@@ -2,16 +2,34 @@
 
 namespace Franzl\Middleware\Whoops;
 
-use Psr\Http\Message\ServerRequestInterface as Request;
+use Franzl\Middleware\Whoops\Insides\AbstractMiddleware;
 use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
 
 /**
  * ErrorMiddleware class for use with Zend's Stratigility middleware pipe
  */
-class ErrorMiddleware
+class ErrorMiddleware extends AbstractMiddleware
 {
+
+    /**
+     * @return ErrorMiddleware
+     */
+    public static function createNewInstance()
+    {
+        return new ErrorMiddleware(WhoopsRunner::createNewInstance());
+    }
```

Comment:
When you return `new static(...)` here, you can move this into the abstract base class.

Also, I'd prefer `newInstance()` for the method name.